src/proto.js

```diff
@@ -377,6 +377,19 @@ exports.retry = function (err) {
   })
 }
 
+/**
+ * Determine if a feature flag is enabled in the set of features
+ * for this integration.
+ *
+ * @param {String} name Name of the feature flag
+ * @return {Boolean}
+ * @api public
+ */
+
+exports.featureOn = function (name) {
+  return this.features.includes(name)
+}
+
```

Comment:
A helper function that I'm not sure is worth adding or renaming. Any thoughts? The alternative is to access features directly and run call include: `this.features.include('my_flag')`

Reply:
An alternative approach would be to generate an object from the feature flag list, then we can use a syntax like this:
```js
if (features.myFlag) {
  ...
}
```

Reply:
Ahh, this is cleaner. I like it. Will solicit more feedback from the pillar before iterating.